sudo_plugin/src/core.rs

```diff
@@ -60,6 +62,7 @@ pub unsafe extern "C" fn open<P: 'static + IoPlugin, S: IoState<P>>(
         plugin_options_ptr,
         plugin_printf,
         conversation,
+        conv_f,
```

Comment:
I'm only constructing a `PrintFacility` outside of the `IoEnv` since I might need to use it if `IoEnv::new` fails. Let's just construct the `ConversationFacility` inside the `IoEnv` like it also does with `PrintFacility`.